lib/cdo/delete_accounts_helper.rb

```diff
@@ -285,6 +285,22 @@ def purge_code_review_comments(user_id)
     @log.puts "Cleared #{comments_count} CodeReviewComment" if comments_count > 0
   end
 
+  def clean_and_destroy_code_reviews(user_id)
+    # anonymize notes the user wrote
+    comments_written = CodeReviewNote.where(commenter_id: user_id)
+    comments_written_count = comments_written.count
+    comments_written.each do |comment|
+      comment.commenter_id = nil
```

Comment:
I see this is how the Jira ticket is written, but surprised we're not deleting the contents of the comment? For future reference, @maureensturgeon could you clarify the reasoning here? Do we need to keep comments for legal reasons or something?

Reply:
This is what was requested by Mike, I think so that if someone looks back at their code review they can still see the comments that they were given